tests_fp/pricing_models/test_calc_in_given_out_fp.py

```diff
@@ -7,7 +7,7 @@
 from typing import TYPE_CHECKING
 
 import numpy as np
-from calc_test_dataclasses import (
+from calc_test_dataclasses_fp import (
```

Comment:
I assume you have to do this in the other pricing model fp tests?

Reply:
I describe it in this PR (https://github.com/element-fi/elf-simulations/pull/346), this is incorporating that change to avoid the error. strangely enough it was only one file. someohow the others didn't create this naming conflict? dunno if it has something to do with caching.

Reply:
Probably just a rebase issue? Both `calc_in_given_out_fp` and `calc_out_given_in_fp` use `calc_test_dataclasses` so make sure both have the rename.

Reply:
oh lol I just scrolled 5 lines down and see that they do. Ok great so it looks like it's all in this PR and the other can be closed 👍 perfect!